src/module.c

```diff
@@ -10385,6 +10388,8 @@ void moduleFireServerEvent(uint64_t eid, int subid, void *data) {
                 moduledata = data;
             } else if (eid == REDISMODULE_EVENT_SWAPDB) {
                 moduledata = data;
+            } else if (eid == REDISMODULE_EVENT_CONFIG) {
```

Comment:
looks like we forgot to document that in the doc comment above RM_SubscribeToServerEvent

Reply:
done